trio/tests/test_signals.py

```diff
@@ -41,6 +41,12 @@ async def test_open_signal_receiver_restore_handler_after_one_bad_signal():
     assert signal.getsignal(signal.SIGILL) is orig
 
 
+async def test_open_signal_receiver_empty_fail():
+    with pytest.raises(AttributeError):
```

Comment:
You can pass `match=...` to `pytest.raises` to make sure you get the expected message. It's not mandatory or anything though